packages/devtools-core/src/App/DevTools/index.tsx

```diff
@@ -53,13 +53,52 @@ const StyledPanel = styled.div`
   max-height: calc(100vh - ${HeaderHeight});
 `;
 
+const RecordButtons = styled.div`
+  position: absolute;
+  bottom: 5px;
+  right: 5px;
+`;
+
+function post({ func, universe, actionLog }: any) {
+  const name = prompt("Name:");
```

Comment:
It'd be nice to have a modal for this (also mentioning what valid test names are, etc)